paper_framework_sources/p2079_system_execution_context.md

```diff
@@ -337,6 +390,61 @@ This would work at task granularity, for each `schedule()` call that we connect
 In either case we can add the priority in a separate paper.
 It is thus not urgent that we answer this question, but we include the discussion point to explain why they were removed from the paper.
 
+## Reference implementation
+
+The authors prepared a reference implementation in [`stdexec`](https://github.com/NVIDIA/stdexec).
+
+A few key points of the implementation:
+* The implementation is divided into two parts: "host" and "backend". The host part implements the API defined in this paper and calls the backend for the actual implementation. The backend provides the actual implementation of the system context.
+* Allows link-time replaceability for `system_scheduler` (link time). Provides examples on doing this.
+* Defines a simple C API between the host and backend parts. This way, one can easily extend this interface when new features need to be added to `system_context`.
+* Uses preallocated storage on the host side, so that the default implementation doesn't need to allocate memory on the heap when adding new work to `system_scheduler`.
+* Guarantees a lifetime of at least the duration of `main()`.
+* As the default implementation is created outside of the host part, it can be shared between multiple binaries in the same process.
+* IN PROGRESS: Provides an example on implementing the `drive()` behavior by replacing the default implementation.
+* TODO: Use OS scheduler for implementation.
+
+## Addressing received feedback
+
+### Allow for system context to borrow threads
+Early feedback on the paper from Sean Parent suggested a need for the system context to carry no threads of its own and to take over the main thread.
```

Comment:
```suggestion
Early feedback on the paper from Sean Parent suggested a need for the system context to support a configuration where it carries no threads of its own and takes over the main thread.
```

This wording is a bit repetitive given the wording at the end of 5.1 in R3, though I appreciate that condensing the feedback makes sense. Maybe it is worth simply removing the feedback related notes elsewhere?